src/main/java/missa/Ui.java

```diff
@@ -0,0 +1,126 @@
+package missa;
+
+import missa.task.Task;
+
+/**
+ * missa.Ui class that deals with interactions with the users.
+ */
+public class Ui {
+    /** An empty line */
+    private String emptyLine = "________________________________________\n";
+
+    /** Greeting sentences */
+    private String greeting = "Hello from Miss A\n"
+            + "What can I do for you?\n"
+            + emptyLine
+            + "I can record 3 types of tasks now.\n"
+            + "ToDos: e.g. todo clean my room\n"
+            + "Deadlines: e.g. deadline submit homework /by 2021-4-1 21\n"
+            + "Events: e.g. event lecture /from 2021-4-2 14 /to 2021-4-2 16\n"
+            + emptyLine
+            + "Here are the commands that I can understand:)\n"
+            + "\"list\": I will display the latest task list.\n"
+            + "\"mark/unmark + number\": I will mark/unmark task in the list!\n"
+            + "\"delete + number\": I will remove task from the list.\n"
+            + "\"find + keyword\": I will search for relevant tasks recorded!\n"
+            + "\"bye\": this program will be closed.\n";
```

Comment:
I like how easy this is to read! Would consider using this style in my code too!